### ISSUE TITLE
In scan mode, the list items present under 'Getting started' are not accessible.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1369]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator
Username: hcltester200@outlook.com/hcltester800@outlook.com

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Turn On Narrator using 'Ctrl+Win+Enter'.
4. Navigate to the 'Get myprofile' button using down arrow key in scan mode.

**Actual Result:**
In scan mode, the list items present under 'Getting started' are not accessible.

**Expected Result:**
In scan mode, the list items present under 'Getting started' should be accessible.

**Observation:**
Issue is not repro with NVDA.

**User Impact:** 
Screen reader users will be impacted as they will not be able to access the specified controls in scan mode limiting their usage of the application.
 
**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.02
